### PR TITLE
tests(ticdc): add table actor integration test

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: ref #3881

this is a part of table pipeline actor model refactor #3881

### What is changed and how it works?
add table actor cdc test

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: tests/integration_tests/_utils/start_tidb_cluster_impl [hunk 1]
```diff
@@ -17,7 +17,7 @@ random_file_name=
 randomGenSocketsConf() {
 	random_str=$(date '+%s%N')
 	if [ "$(uname)" == "Darwin" ]; then
-		random_str=$(cat /dev/random | LC_CTYPE=C tr -dc "a-zA-Z0-9" | head -c 10)
+		random_str=$(cat /dev/random | LC_ALL=C tr -dc "a-zA-Z0-9" | head -c 10)
```
Comment (hi-rustin) [inline tests/integration_tests/_utils/start_tidb_cluster_impl:20]: Why we need to change this?

File: tests/integration_tests/bank_table_actor/run.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+set -eu
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=cdc.test
+SINK_TYPE=$1
+
+function prepare() {
+	if [ "$SINK_TYPE" == "kafka" ]; then
```
Comment (hi-rustin) [inline tests/integration_tests/bank_table_actor/run.sh:14]: Is this a redundant judgment?

File: tests/integration_tests/_utils/start_tidb_cluster_impl (hunk 1 above)
Comment (sdojjy) [reply tests/integration_tests/_utils/start_tidb_cluster_impl:20]: to make  sure `tr` command can be executed.
```
cat /dev/random | LC_CTYPE=C tr -dc "a-zA-Z0-9" | head -c 10
tr: Illegal byte sequence

```

**Review by hi-rustin — APPROVED** [review]

File: tests/integration_tests/bank_table_actor/run.sh (hunk 2 above)
Comment (hi-rustin) [reply tests/integration_tests/bank_table_actor/run.sh:14]: We need to remove a judgment.

Comment (sdojjy) [reply tests/integration_tests/bank_table_actor/run.sh]: removed

**Review by overvenus — APPROVED** [review]

**Pull request merged**